src/wdio/services/TerraService.js

```diff
@@ -67,26 +67,37 @@ const beAccessible = (options) => {
 * @property {Object} customProperties - An object containing the CSS custom properties to assert.
 * @property {Object} options - Additional options for testing.
 */
-const themeEachCustomProperty = (customProperties, options = {}) => {
-  Object.entries(customProperties).forEach(([key, value]) => {
+const themeEachCustomProperty = (customSelector, customProperties) => {
+  let selector = global.browser.options.terra.selector;
+  let styleProperties = {};
```

Comment:
you could technically make this always equal to the last argument
```js
// If more than 1 argument, selector is fist 
const selector = arguments.length > 1 ? arguments[0] : global.browser.options.terra.selector;
// Style properties are always last.
const styleProperties = arguments[arguments.length - 1];
```

Then if arguments.l

Reply:
When I run that with the wdio testing interface, I see that it's actually setting arguments to something else.

![screen shot 2018-01-23 at 11 42 15 am](https://user-images.githubusercontent.com/2552573/35291286-7d2b91c2-0032-11e8-863d-9f3eaef5433f.png)

Do you know if there is away around that with the wdio command line? If not, we may not be be able to rely on the arguments object.

Reply:
Sorry, arrow functions don't have arguments. Use rest params

```js
// Example
const x = (...args) => console.log(args)
```

Reply:
Forgot all about that myself! Thanks for the reminder! It works great, and even did a little extra cleanup too :) https://github.com/cerner/terra-toolkit/pull/48/commits/1ecb511eac01274dd31faa17e2eb9597c7665788